src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -86,10 +86,10 @@ public Socket(SocketType socketType, ProtocolType protocolType)
         // Initializes a new instance of the Sockets.Socket class.
         public Socket(AddressFamily addressFamily, SocketType socketType, ProtocolType protocolType)
         {
-            s_loggingEnabled = Logging.On;
+            s_loggingEnabled = SocketsEventSource.Log.IsEnabled();
             if (s_loggingEnabled)
             {
-                Logging.Enter(Logging.Sockets, this, "Socket", addressFamily);
+                NetEventSource.Enter(this, "Socket", addressFamily, NetEventSource.Component.Socket);
```

Comment:
Would be easier if you moved the Component type first as the transformation will be 1:1:  Logging.Sockets -> NetEventSource.Component.Socket; Logging->NetEventSource and the rest remains the same.